### DISCUSSION TITLE
convert json to Datatyp at runtime, datatyp to convert at compiler time is unknown ( help)

### DISCUSSION THREAD

**Discussion by bnadem** [nlohmann/json#3869] [Ideas]
i become data json, i have no idea about datatype at compiler time
i must to convert to typedef struct or union maybe nested structure at runtime

{
"type":"struct"
"name:"MyStruct"
"struct":  {
       "var1":"int",
        "var2":float;
       "var3":"double"
}
}

convert AT RUNTIME to

typedef struct {
 var1:int;
 var2:float;
 var3:double }MyStruct`

**Comment by nlohmann** [comment]
Have you looked at https://json.nlohmann.me/features/arbitrary_types/ ?
